### DISCUSSION TITLE
The reason of recommendation of "For you" is wrong

### DISCUSSION THREAD

**Discussion by cshuaimin** [community/community#19669] [Other Feature Feedback, Questions, & Ideas]
I did not star this repository (I'm pretty sure this is the first time I've seen it), but the popup shows it's because I starred the repository.
<img width="916" alt="截屏2022-06-29 13 48 04" src="https://user-images.githubusercontent.com/24775746/176361430-91d655e9-eab3-4859-99d7-5c84c1dc11d2.png">

**Comment by melvincwng** [comment]
你好 @cshuaimin ，你能做到以下几点：

 1. 转到该特定存储库并检查您是否已加注星标。
 2. 如果你没有给它加星标，而它说你有星标，我猜这可能是一个潜在的错误。
 3. 这种情况可以向 GitHub Bug Report - https://support.github.com/contact/bug-report 报告。

 希望这可以澄清并有所帮助😄

 如果这有帮助，请“标记为答案”并在 GitHub 上关注我。

 谢谢你，祝你有美好的一天。
